Zend/tests/return_types/void_disallowed1.phpt

```diff
@@ -9,4 +9,4 @@ function foo(): void {
 
 // Note the lack of function call: function validated at compile-time
 --EXPECTF--
-Fatal error: A void function must not return a value (did you mean "return;" instead of "return null;"?) in %s on line %d
+Fatal error: Function foo() with return type void must not return a value (did you mean "return;" instead of "return null;"?) in %s on line %d
```

Comment:
I'm wondering if the run-time return type error message format should be changed to
`Method Foo::bar() must return type int, array returned` or
`Method Foo::bar() must have a return value of type int, array returned`.